express-hello-world/index.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const app = express();
+
+app.use(function (req, res, next) {
+  res.send('To infinity and beyond!');
+  next();
```

Comment:
It's usually not a good idea to call `next()` if there's no additional middleware you need to run for the same request.